css/src/components/popover.scss

```
@@ -19,31 +23,81 @@ $popover-width: 224px !default;
 	.popover-content {
 		position: absolute;
 		width: $popover-width;
-		margin-block-start: $spacer-3;
 		padding: $spacer-5;
+		transform: none;
+		inset-inline-start: 0;
 		border: $popover-border;
 		border-radius: $popover-border-radius;
 		background-color: $popover-background-color;
 		box-shadow: $popover-shadow;
+
+		// hide popover until after position is calculated, then show
+		visibility: hidden;
 		z-index: $zindex-popover;
 	}
 
-	&.popover-right {
+	&.popover-top {
 		.popover-content {
-			inset-inline-end: 0;
+			inset-block-end: 100%;
+			margin-block-end: $spacer-3;
 		}
 	}
 
-	&.popover-center {
+	&.popover-caret {
 		.popover-content {
-			inset-inline-start: -25%;
+			margin-block-start: 0;
+			overflow: visible;
+
+			&::before {
+				@include chevron-up;
+
+				position: absolute;
+				inset-inline-start: var(--caret-left, 50%);
+				inset-inline-end: auto;
+				inset-block-start: calc($chevron-arrow-size / -2);
```

```suggestion
				inset-block-start: calc($chevron-arrow-size / -1.75);
```
Makes the caret border more flush with the popover content border